src/watchdog/observers/fsevents.py

```diff
@@ -87,7 +87,8 @@ def queue_events(self, timeout, events):
 
         while events:
             event = events.pop(0)
-            logger.info(event)
+            flags = ", ".join(attr for attr in dir(event) if getattr(event, attr) is True)
+            logger.info(f"{event}: {flags}")
```

Comment:
Perhaps should this be logged at the DEBUG level level only, and protected like:
```suggestion
            if logging.getEffectiveLevel() >= logging.DEBUG:
                flags = ", ".join(attr for attr in dir(event) if getattr(event, attr) is True)
                logger.debug(f"{event}: {flags}")
```

Reply:
Oh right, I forgot demoting all the logger calls to debug in my previous PR. It's certainly possible and more correct to lower these c alls to debug, but that means the tests will produce a ton more log output. That is, unless py.test can be configured to only capture specific loggers instead of all of them. I had only found `--log-cli-level` and thus ended up using `INFO` to have _somewhat_ readable log output.

Reply:
Good point. I have now changed all log messages from within `queue_events` to debug level. I've also batched the log messages for native events at the beginning. Since the additional iteration is protected by `logger.getEffectiveLevel` there should not be a performance impact. This has the advantage that we can see which events were emitted for a single callback.